src/brogue/Math.c

```diff
@@ -166,9 +167,15 @@ int rand_range(int lowerBound, int upperBound) {
 
 // seeds with the time if called with a parameter of 0; returns the seed regardless.
 // All RNGs are seeded simultaneously and identically.
-unsigned long seedRandomGenerator(unsigned long seed) {
+uint64_t seedRandomGenerator(uint64_t seed) {
     if (seed == 0) {
-        seed = (unsigned long) time(NULL) - 1352700000;
+        seed = (uint64_t) time(NULL) - 1352700000;
+        // bit mixing (from MurmurHash3)
+        seed ^= seed >> 33;
+        seed *= 0xff51afd7ed558ccd;
+        seed ^= seed >> 33;
+        seed *= 0xc4ceb9fe1a85ec53;
+        seed ^= seed >> 33;
```

Comment:
It's essentially cosmetic. I wanted auto-generated seeds to "jump around" instead of increasing monotonically, which gives the impression than they aren't really random. But it works just fine without it. Shall I remove it?

Reply:
I think people expect increasing seeds, I don't see any reason to change it really

Reply:
Will revert, then 👍